output/dot/dot.go

```diff
@@ -72,7 +72,7 @@ func (d *Dot) OutputSchema(wr io.Writer, s *schema.Schema) error {
 		"Schema":          s,
 		"showComment":     d.config.ER.Comment,
 		"showDef":         !d.config.ER.HideDef,
-		"hideColumnTypes": d.config.ER.HideColumnTypes,
+		"showColumnTypes": d.config.ER.ShowColumnTypes,
```

Comment:
This line is not necessary because it is not used in the template file.
(also `output/mermaid/mermaid.go` and `output/plantuml/plantuml.go`)

Reply:
Thank you!